### PR TITLE
chore: Misc Plugin fixes and updates

### PR DESCRIPTION
**Pull request by tfhartmann**
Update @types/react and @types/react-dom to version 18

Fix Pagerduty plugin errors

Misc plugin updates to resolve depend errors 

update container build to use build pack

---

## Write Good Commit Messages

[The seven rules of a great Git commit message](https://cbea.ms/git-commit/)

* Separate subject from body with a blank line
* Limit the subject line to 50 characters
* Capitalize the subject line
* Do not end the subject line with a period
* Use the imperative mood in the subject line
* Wrap the body at 72 characters
* Use the body to explain what and why vs. how

### CODE REVIEW

**Comment by tfhartmann** [comment]
atlantis plan

**Comment by tfhartmann** [comment]
atlantis apply -p dev

File: docs/templates.md [hunk 1]
```diff
@@ -4,3 +4,28 @@ List of actions https://backstage.broadinstitute.org/create/actions
 
 Interactive Editor for Software Templates
 https://backstage.broadinstitute.org/create/edit
+
+Local Development of Software Templates
+
+Becase we are breaking up software templates into reusable components, we have to use the `url` field in the app-config.yaml to reference the templates. This means that when you are developing a new template, you need to push your changes to a branch in order to test them. You can also create a monolithic template that includes all the components, and then split the config into separate files once you are happy with the result.
```
Comment (coreone) [inline docs/templates.md:10]: Because

**Review by coreone — CHANGES_REQUESTED** [review]

File: docs/templates.md (hunk 1 above)
Comment (tfhartmann) [reply docs/templates.md:10]: oh yikes, I forgot that got checked in! :doh:

Comment (tfhartmann) [reply docs/templates.md]: fixed

**Review by coreone — APPROVED** [review]

**Comment by tfhartmann** [comment]
atlantis apply

**Pull request merged**